docs/groovy-script/mods/abyssaltweaker/ritual.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Necronomicon Ritual"
+titleTemplate: "Abyssal Tweaker | CleanroomMC"
+description: "Allows setting up rituals that can be executed with various tiers of Necronomicon."
+source_code_link: "https://github.com/TeamDimensional/AbyssalTweaker/blob/master/src/main/java/com/teamdimensional/abyssaltweaker/compat/groovyscript/Ritual.java"
+---
+
+# Necronomicon Ritual (Abyssal Tweaker)
+
+## Description
+
+Allows setting up rituals that can be executed with various tiers of Necronomicon.
+
+:::::::::: details Warning {open id="warning"}
+For technical reasons the Ritual compatibility will not validate whether the ritual is executed in a valid dimension. By default only Overworld and Abyssalcraft's dimensions are valid, and other dimensions have to be added through the config file.
```

Comment:
could this specify that the `validate` here refers specifically to registering the ritual, not to performing the ritual.

Reply:
oops yeah